### PR TITLE
Fix: flow error in UploadStateContent

### PR DESCRIPTION
**Pull request by wenboyu2**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @wenboyu2 has signed the CLA. Thanks for the pull request!

File: src/components/ContentUploader/UploadStateContent.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ const UploadStateContent = ({ message, inputLabel, useButton = false, onChange }
     const messageContent = message ? <div className='bcu-upload-state-message'>{message}</div> : null;
     const inputLabelClass = useButton ? 'btn btn-primary be-input-btn' : 'be-input-link';
 
-    const handleChange = (event: Event<*>) => {
+    const handleChange = (event: Event) => {
+        if (!onChange) {
+            return;
+        }
+
         onChange(event);
```
Comment (priyajeet) [inline src/components/ContentUploader/UploadStateContent.js]: This will be a react synthetic event. Do we want that or the original event?

Comment (priyajeet) [reply src/components/ContentUploader/UploadStateContent.js]: btw dunno if you saw https://github.com/box/box-ui-elements/pull/297/files#r186877887

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #295

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentUploader/UploadStateContent.js (hunk 1 above)
Comment (wenboyu2) [reply src/components/ContentUploader/UploadStateContent.js]: yeah Jeremy pointed to me your comment. updated

**Pull request merged**